### ISSUE TITLE
[Bug]: sp-split-button more button should be labelled by button element

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#3305]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

sp-split-button

### Expected behavior

The popup button part of the split button should be labelled by the button part

### Actual behavior

It is currently labelled a hardcoded "More"

### What browsers are you seeing the problem in?

Firefox, Chrome, Safari, Microsoft Edge

### How can we reproduce this issue?

1. Go to https://opensource.adobe.com/spectrum-web-components/components/split-button/#sizes
2. Right Click on the More button and inspect the Accessibility Properties

**Comment by jnurthen** [comment]
Having the same label isn't an issue as one will be read as a menu button (or a button with a popup) and one is a simple button.
